testings/test_open.py

```diff
@@ -49,6 +51,32 @@
         )
 
 
+def assert_file_not_in_use(filename):
+    platform = sys.platform
+    if platform == "linux":
+        from subprocess import check_output,Popen, PIPE
+
+        lsout = Popen(['lsof',filename],stdout=PIPE, shell=False)
+        check_output(["grep",filename], stdin=lsout.stdout, shell=False)
+
+        for proc in psutil.process_iter():
+            try:
+                for item in proc.open_files():
+                    if filename == item.path:
+                        return True
+            except Exception:
+                pass
+        raise Exception("File is in use.")
```

Comment:
@willberriss

Here is the line where I tried to make the test fail, if the file is in use.

Reply:
I found it! I did opposite.